doc/usecases/elektra_web/UC_remove_cluster.md

```diff
@@ -0,0 +1,19 @@
+# Use Case: Remove cluster
```

Comment:
clusterd actually manages all clusters, I don't think it makes much sense to have recursion of clusterd's, because there are many issues with that approach (see @Namoshek's comment) It would also make the setup and maintainance a lot harder.